src/components/places/map-with-all-places/map-with-all-places.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+const { withScriptjs, withGoogleMap, GoogleMap } = require("react-google-maps")
+const { MarkerWithLabel } = require("react-google-maps/lib/components/addons/MarkerWithLabel")
+
+const MapWithAllPlaces = ({ places }) => {
+    return (
```

Comment:
Jak nie robisz nic więcej tylko zwracasz to możesz zapisać to tak:
```
const MapWithAllPlaces = ({ places }) => (
  <GoogleMap>
  {...}
)
```

Reply:
a fakt, chociaz lubie miec returna, ale trzeba sie przyzwyczajac do dobrych praktyk XD

Reply:
Heheh nom. Ja już jestem tak przyzwyczajony, że od razu zauważam i mnie wkurza ;P